### DISCUSSION TITLE
Is it possible to resolve symlinks during the builder image's `COPY`ing of the source code?

### DISCUSSION THREAD

**Discussion by hab25** [buildpacks/community#88] [Q&A]
My source code directory has a symlink that points to a path not contained in the directories' subtree. This is because I have a monorepo, and I share code between projects with symlinks. Therefore, when I try building, the symlink becomes dead (dangling) after being copied into the builder.

A solution that would work is to resolve the symlink as the builder image's `COPY`ing of my source code happens; is this possible?

**Comment by jromero** [comment]
What tooling are you using to run buildpacks? Is it `pack` or via a different method?

**Reply by hab25** [reply]
Yes, `pack`.
